examples/00_quick_start/fastai_movielens.ipynb

```diff
@@ -432,7 +432,7 @@
             ],
             "source": [
                 "with Timer() as train_time:\n",
-                "    learn.fit_one_cycle(EPOCHS, max_lr=5e-3)\n",
+                "    learn.fit_one_cycle(EPOCHS, lr_max=5e-3)\n",
```

Comment:
Come on! This is the kind of breaking change that should be announced somehow. I hope that at least the error tell you that you have to change the name?